controlplane/control/sampler.go

```diff
@@ -1,11 +1,16 @@
 package control
 
 import (
+	"errors"
 	"fmt"
+	"regexp"
 
 	"github.com/neblic/platform/controlplane/protos"
 )
 
+var uidValidationRegex = regexp.MustCompile(`^[\.\/\ ()\w-]*$`)
+var uidValidationErrTemplate = "invalid %s uid %s, expected alphanumerical with spaces and ./() characters"
```

Comment:
* Should we allow spaces? I think it is dangerous to have an uid with spaces
* The comment doesn't say `-` and `_` are allowed.

Reply:
- True, will remove the spaces
- I just updated the description to add those characters (and remove the spaces)